ext/gmp/gmp.c

```diff
@@ -862,6 +862,27 @@ static inline void _gmp_unary_opl(INTERNAL_FUNCTION_PARAMETERS, gmp_unary_opl_t
 }
 /* }}} */
 
+static bool gmp_verify_base(zend_long base)
+{
+	if (base && (base < 2 || base > GMP_MAX_BASE)) {
+		zend_argument_value_error(2, "must be between 2 and %d", GMP_MAX_BASE);
```

Comment:
I might prefer to pass the argument number as a parameter, although it seems like it is only used when the base argument is in second position